recipe/build.sh

```diff
@@ -107,6 +107,11 @@ if [[ -d $PREFIX/lib_pypy ]]; then
 	pypy -m lib2to3.pgen2.driver lib-python/3/lib2to3/PatternGrammar.txt
 	popd
 else
+	# Make sure the site-packages dir match with cpython
+	mkdir -p $PREFIX/lib/python${PY_VERSION}/site-packages
+	mv $PREFIX/lib/pypy${PY_VERSION}/site-packages/README $PREFIX/lib/python${PY_VERSION}/site-packages/
+	rm -rf $PREFIX/lib/pypy${PY_VERSION}/site-packages
+	ln -sf $PREFIX/lib/python${PY_VERSION}/site-packages $PREFIX/site-packages/lib/pypy${PY_VERSION}
```

Comment:
Would this be simpler as
```suggestion
	# Add a symlink for no-arch installations
	ln -sf $PREFIX/lib/pypy${PY_VERSION}/site-packages $PREFIX/site-packages/lib/python${PY_VERSION}
```

Reply:
No that doesn't work: `$PREFIX/lib/python$PY_VERSION` does not exist so you cannot create a `LINK_NAME` there.